### ISSUE TITLE
The window's client area becomes completely white when the mouse pointer enters it

### ISSUE THREAD

**Issue by ghost** [zulip/zulip-desktop#1326]
**Describe the bug**

The whole client area of the window becomes white when the mouse pointer *enters* the window's client area.

**To Reproduce**

* Use version 5.10.1 (I installed it by unpacking from `Zulip-5.10.1-x64.tar.xz`).
* Start Zulip with the mouse pointer outside of the area where the Zulip window will appear. Zulip starts up fine and shows everything as expected. 
* Move the mouse pointer into the client area of the Zulip window and the problem appears: the whole client area of the window becomes white. Only the  menu is visible, can be accessed and used to close Zulip.

If the mouse pointer is already in the window's client area on Zulip startup, the problem starts after the mouse pointer been moved out of and back into the window's client area. 

**Expected behavior**

Show the contents of the window.
This problem didn't happen in version 5.10.0.

**Desktop (please complete the following information):**

- Operating System: Xubuntu 22.04

**Additional context**

Output when started from the terminal:

This appears immediately on startup:
```
Reverting GDK_BACKEND to work around https://github.com/electron/electron/issues/28436
```

This appears a few seconds after the problem occured, and then repeatedly again after a few seconds:
```
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94495)
    at WebContents.send (node:electron/js2c/browser_init:2:79721)
    at send (/opt/zulip-5.10.1/resources/app.asar/dist-electron/index.js:33239:12)
    at Timeout._onTimeout (/opt/zulip-5.10.1/resources/app.asar/dist-electron/index.js:43640:7)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
```

**Comment by Rusty-Weasel** [comment]
looks like the same as https://github.com/zulip/zulip-desktop/issues/1327#issuecomment-1719619195 , is allready fixed,
try the new released 5.10.**2**

**Comment by ghost** [comment]
Yes, 5.10.2 fixes the problem.
